CONTRIBUTING.md

```diff
@@ -4,10 +4,10 @@ You can add a record using any of the following three methods. We have a few met
 
 - Complete PR
   - add record in [README](https://github.com/opensourcecities/montreal/blob/master/README.md) AND
-  - add record's `yml` file (based on [Type of Records](#type-of-records)) in `data` folder
+  - add record's `yml` file (based on [Type of Records](#type-of-records)) in `content` folder
```

Comment:
Using `yml` in `data` folder was the old way, and now it's changed to `md` file in `content` folder, can you adjust file format in your PR as well (`yml` -> `md`)?

Reply:
@khos2ow Can you check d831737ab0 ? It is still yml !

Reply:
That's fine. Your description was good. The base file is in `.md` but the metadata on top is in `yml`.